spec/system/ocr_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'OCR', :paperclip_processing, :sidekiq_inline do
+  include ProfileStories
+
+  subject { page }
```

Comment:
Super minor style thing - in these feature/system specs I would just not bother with `subject` at all -- seeing `page` right in the examples is very readable. I feel 1% strong about this, ignore if you disagree.